packages/components/src/entities/SampleOverviewPanel.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC, memo, useCallback, useMemo, useState } from 'react';
+
+import { QueryConfig, QueryModel } from '../public/QueryModel/QueryModel';
+import { Actions } from '../public/QueryModel/withQueryModels';
+import { User } from '../internal/components/base/models/User';
+import { Container } from '../internal/components/base/models/Container';
+import { LoadingPage } from '../internal/components/base/LoadingPage';
+
+import { getSampleStatusType, isSampleOperationPermitted } from '../internal/components/samples/utils';
+import { ALIQUOT_FILTER_MODE, SampleOperation } from '../internal/components/samples/constants';
+import { isELNEnabled } from '../internal/app/utils';
+import { getContainerFilterForLookups } from '../internal/query/api';
+
+import { SampleAliquotsSummary } from './SampleAliquotsSummary';
+import { SampleDetailEditing } from './SampleDetailEditing';
+
+import { getSampleAuditBehaviorType } from './utils';
+import { SampleStorageLocation } from './models';
+import { useSampleTypeAppContext } from './SampleTypeAppContext';
+
+interface Props {
+    SampleStorageLocationComponent: SampleStorageLocation;
+    actionChangeCount: number;
+    actions: Actions;
+    canUpdate: boolean;
+    noun?: string;
+    onDetailUpdate: (skipChangeCount?: boolean) => void;
+    sampleContainer: Container;
+    sampleModel: QueryModel;
+    title: string;
+    user: User;
+}
+
+export const SampleOverviewPanel: FC<Props> = memo(props => {
+    const {
+        canUpdate,
+        sampleContainer,
+        title,
+        onDetailUpdate,
+        sampleModel,
+        actionChangeCount,
+        actions,
+        user,
+        noun,
+        SampleStorageLocationComponent,
+    } = props;
+    const { getWorkflowGridQueryConfigs, ReferencingNotebooksComponent, detailRenderer } = useSampleTypeAppContext();
+    const [isEditing, setIsEditing] = useState(false);
+    const row = useMemo(() => {
+        return !sampleModel.isLoading ? sampleModel.getRow() : undefined;
+    }, [sampleModel]);
+
+    const auditBehavior = getSampleAuditBehaviorType();
+    const sampleStatusType = getSampleStatusType(row);
+    const _canUpdate = !isEditing && canUpdate;
+    const aliquotedFrom = sampleModel.getRowValue('AliquotedFromLSID/Name');
+    const sampleId = sampleModel.getRowValue('RowId');
+    const sampleLsid = sampleModel.getRowValue('LSID');
+    const sampleName = sampleModel.getRowValue('Name');
+    const sampleType = sampleModel.schemaQuery.queryName;
+    const isAliquot = aliquotedFrom != null;
+    const { isMedia } = sampleModel.queryInfo;
+    const jobConfigs = getWorkflowGridQueryConfigs(
```

Comment:
nit: There is a decent chance to optimize here so we're not creating these configs on each render cycle. Mainly, this can depend on the `isAliquot` bit to align with when it is needed by the underlying components. Something like:

```ts
const aliquotJobsQueryConfig = useMemo((): QueryConfig => {
    if (isAliquot) {
        const queryConfigMap = getWorkflowGridQueryConfigs(
            ['all'], // i.e. ALL_JOBS_QUEUE_KEY
            'sample-aliquot-jobs',
            null,
            null,
            null,
            sampleLsid,
            null,
            ALIQUOT_FILTER_MODE.aliquots
        );
        return Object.values(queryConfigMap)[0];
    }

    return undefined;
}, [getWorkflowGridQueryConfigs, isAliquot, sampleLsid]);
```

Subsequently:

```tsx
{!isAliquot && (
    <div className="col-xs-12 col-md-4">
        <SampleAliquotsSummary
            sampleId={sampleId}
            sampleSet={sampleType}
            sampleLsid={sampleLsid}
            sampleRow={row}
            sampleSchemaQuery={sampleModel.queryInfo.schemaQuery}
            aliquotJobsQueryConfig={aliquotJobsQueryConfig}
        />
    </div>
)}
```

Reply:
Makes sense (except the conditional needed to be flipped to `if (!isAliquot) {`). Thanks for the suggestion. Patch applied.